lib/socket.ts

```diff
@@ -112,6 +113,10 @@ export function createSocket(options: ConnectionOptions): Promise<HaWebSocket> {
           socket.removeEventListener("close", closeMessage);
           socket.removeEventListener("error", closeMessage);
           socket.haVersion = message.ha_version;
+          if (atLeastHaVersion(socket.haVersion, 2022, 9, 0)) {
```

Comment:
you can leave `patch` undefined. I think that otherwise it will not work in dev. `Number("0.dev0")` results in NaN

Reply:
https://github.com/home-assistant/home-assistant-js-websocket/blob/master/lib/util.ts#L59

```suggestion
          if (atLeastHaVersion(socket.haVersion, 2022, 9)) {
```